wrenidm-workflow-flowable/src/main/java/org/wrensecurity/wrenidm/workflow/flowable/impl/FlowableServiceImpl.java

```diff
@@ -153,7 +153,7 @@ public class FlowableServiceImpl implements RequestHandler {
 
     @Reference(
             service = DataSourceService.class,
-            cardinality = ReferenceCardinality.MULTIPLE,
+            cardinality = ReferenceCardinality.AT_LEAST_ONE,
```

Comment:
This is probably not correct as JDBC repo service can not exist without data source, but workflow service can exist without one. The main difference is that JDBC repo service is just one type of repo service (previously we had OrientDB and there were activities to add LDAP based repository support).

However as this solves the original issue for now, I will probably accept this PR. The correct solution would probably require a bit more significant rewrite of the bootstrap logic in this class.

Reply:
Or alternatively we can simply drop support for the embedded H2 in this service and make dataSource mandatory in a sense that if someone want to use embedded H2, they need to configure it as data source service.

Reply:
Support for embedded H2 has been completely removed.